<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>487</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>487</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cost</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>471</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ActualDay">
     <attribute name="title">
      <string>Aktualny dzień</string>
     </attribute>
     <widget class="QTableWidget" name="TodayTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>201</width>
        <height>381</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Produkt</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cena</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="AddButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DeleteRow">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>170</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuń</string>
      </property>
     </widget>
     <widget class="QLabel" name="Suma">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>240</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Suma:</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="CostInCurrentMonth">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>240</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="History">
     <attribute name="title">
      <string>Historia</string>
     </attribute>
     <widget class="QComboBox" name="MounthHistory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>421</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="HistoryTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>211</width>
        <height>341</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>211</width>
        <height>0</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>Produkt</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cena</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="lebelHistory">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>240</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Suma:</string>
      </property>
     </widget>
     <widget class="QLabel" name="priceHistory">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>240</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>9</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>6</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Miesiąc w którym dodajesz:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
